src/chunk.c

```diff
@@ -144,13 +144,13 @@ ChunkResult Uncompressed_UpsertSample(UpsertCtx *uCtx, int *size) {
     return CR_OK;
 }
 
-ChunkIter_t *Uncompressed_NewChunkIterator(Chunk_t *chunk, bool rev) {
+ChunkIter_t *Uncompressed_NewChunkIterator(Chunk_t *chunk, int options) {
     ChunkIterator *iter = (ChunkIterator *)calloc(1, sizeof(ChunkIterator));
     iter->chunk = chunk;
-    if (rev == false) { // iterate from first to last
-        iter->currentIndex = 0;
-    } else { // iterate from last to first
+    if (options & CHUNK_ITER_OP_REVERSE) { // iterate from last to first
```

Comment:
```suggestion
    iter->currentIndex = (options & CHUNK_ITER_OP_REVERSE) ? iter->chunk->num_samples - 1 : 0;
```

Reply:
I generally avoid one-liners ifs as they are more complex to read and comprehend when reading complex code bases.